src/form.rs

```diff
@@ -39,25 +39,42 @@ use crate::{
     meta_type::MetaType,
 };
 
+#[cfg(all(feature = "std", feature = "schema"))]
+use schemars::JsonSchema;
 #[cfg(feature = "serde")]
 use serde::Serialize;
 
+/// Trait to support derivation of `JsonSchema` for schema generation.
+#[cfg(all(feature = "std", feature = "schema"))]
```

Comment:
Turns out that `schemars` indeed can only be used with `std`. Trying to compile it with no-std throws errors like:
```
error[E0433]: failed to resolve: use of undeclared crate or module `std`
   --> src/ty/mod.rs:657:63
    |
657 | #[cfg_attr(all(feature = "serde", feature = "schema"), derive(schemars::JsonSchema))]
    |                                                               ^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `std`
    |
    = note: this error originates in the derive macro `schemars::JsonSchema` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
Indeed it does. Did you try adding `"std"` into the `schema` feature in the manifest?

Reply:
Nope. I thought it is better that a developer explicitly enables `std` feature.

Reply:
Why? `schema` depends on `std` being enabled.

Reply:
okay, I don't have a strong opinion on that. I'll change it now